drake/systems/test/system_test_util.h

```diff
@@ -0,0 +1,53 @@
+#ifndef DRAKE_SYSTEMS_TEST_SYSTEM_TEST_UTIL_H_
+#define DRAKE_SYSTEMS_TEST_SYSTEM_TEST_UTIL_H_
+
+#include "drake/systems/LinearSystem.h"
+#include "drake/systems/cascade_system.h"
+#include "drake/systems/feedback_system.h"
+#include "drake/systems/test/system_test_util.h"
+
+namespace Drake {
+namespace system_test {
+
+template <int StatesAtCompileTime, int InputsAtCompileTime,
+          int OutputsAtCompileTime>
+std::shared_ptr<AffineSystem<EigenVector<StatesAtCompileTime>::template type,
+                             EigenVector<InputsAtCompileTime>::template type,
+                             EigenVector<OutputsAtCompileTime>::template type>>
+CreateRandomAffineSystem(size_t num_states, size_t num_inputs,
```

Comment:
why put this in a shared header if there is only one consumer for now?  is that premature generalization?